README.md

```diff
@@ -85,9 +85,9 @@ ETI ni2http
 
 ETI-NI streams from terrestrial DAB ensembles can also be created by [eti-stuff](https://github.com/JvanKatwijk/eti-stuff) or [dabtools](https://github.com/Opendigitalradio/dabtools). For satellite feeds see above.
 
-    usage: ./ni2http [--list] [--delay] [-i <inputfile>] [-c <config_file>]
+    usage: ./ni2http [--list] [--delay] [-i <inputfile>] [-c <config_file>] [-s <SID>]
 
-Use `--list` option to find SIDs and station names of the streams inside ETI.
+Use `--list` option to find SIDs and station names of the streams inside ETI. If you wish to write stream to stdout, then use `ni2http --sid <SID>`.
```

Comment:
Hi,

Expand the description to indicate that the config_file isn't needed when using the `-s` parameter. For example:

- Use `--list` option to find SIDs and station names of the streams inside ETI.
- If you wish to write stream to STDOUT, then use `ni2http --sid <SID>`. In this case the `config_file` isn't needed.